app/api/orders.py

```diff
@@ -104,6 +104,14 @@ def before_create_object(self, data, view_kwargs):
             raise ConflictException({'pointer': '/data/attributes/amount'},
                                     "Amount cannot be null for a paid order")
 
+        if data.get('payment_mode') in ['stripe', 'paypal', 'alipay', 'omise'] and not data.get('is_billing_enabled'):
+            raise ConflictException({'pointer': '/data/attributes/is_billing_enabled'},
+                                    "Billing information is mandatory for paid orders")
+        if data.get('is_billing_enabled') and not (data.get('company') and data.get('address') and data.get('city') and
+                data.get('zip') and data.get('country')):
+            raise ConflictException({'pointer': '/data/attributes/is_billing_enabled'},
+                                    "Billing information incomplete")
```

Comment:
Use the same above message for this too

Reply:
Actually, both the checks imply different thing and so they have different messages.
In the above case - The user hasn't ticked the checkbox `Enable Billing Information` on the order page while placing a paid order.
While in the second one, the user have checked the checkbox but one or more information is missing. @kushthedude